versioned_sidebars/version-2.5-sidebars.json

```diff
@@ -1249,6 +1249,7 @@
                 }
               ]
     },
-        "contribute-to-rancher"
+        "contribute-to-rancher",
+    "glossary"
```

Comment:
```suggestion
        "contribute-to-rancher",
        "glossary"
```

Reply:
Updated in https://github.com/rancher/rancher-docs/pull/1421/commits/25450ce3f083793b899f5bcf8bf1fa0e93fc17e3.